content/it/docs/home/_index.md

```diff
@@ -0,0 +1,58 @@
+---
+approvers:
+- chenopis
```

Comment:
@jimangel, should we change approvers while doing translations?

Reply:
I would, unless you know the existing approver speaks the target language.